build.gradle

```diff
@@ -185,6 +185,7 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version:'0.0.4', withoutSpringCloudContext
 
   compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.1.3.RELEASE'
+  compile group: 'com.microsoft.azure', name: 'azure-servicebus', version: '1.2.15'
```

Comment:
version 3.0.0 is available

Reply:
wow! i thought i had this in my watchlist as i was tracking for memory leak problem. to be applied in processor/orchestrator - everything ready for v2 jump and now v3 😮. one more thing to investigate this(?) evening

Reply:
when 3.0.0 used
Dependency-Analyze Failure:
16:03:29    One or more dependencies were identified with vulnerabilities that have a CVSS score greater then '0.0': CVE-2018-1000844, CVE-2018-1000850, CVE-2018-20200
16:03:29    See the dependency-check report for more details.

Reply:
this dependency jump can be reviewed later